src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -42,6 +42,21 @@ public class LanguageServer
 
         private TaskCompletionSource<bool> serverCompletedTask;
 
+        private static string checkHelpScript = @"
```

Comment:
We want to use a constant here since `string` can be a constant and this won't change.

```csharp
private const string CheckHelpScript = @"";
```

<sup>(**Note:** There no real guidance for casing of *private* constants so I usually go with the same casing of public constants which is typically Pascal.  Unless it's an interop constant, then keep casing of the original)</sup>

For order, I typically sort fields by access first, then I do:

1. Constants
1. Static readonly
1. Static
1. Instance readonly
1. Instance

Reply:
Changed casing. 
Moved to top of class.